torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -1485,10 +1486,15 @@ def _get_buffers_and_dtypes_for_computation(
         root_module
     )
     for fsdp_state, fsdp_module in zip(reversed(fsdp_states), reversed(fsdp_modules)):
-        for buffer in fsdp_module.buffers():
+        for buffer_name, buffer in fsdp_module.named_buffers():
             if buffer in visited_buffers:
                 continue
             visited_buffers.add(buffer)
+            if (
+                hasattr(fsdp_state, "_ignored_buffer_names")
```

Comment:
Do we need the `hasattr(fsdp_state, "_ignored_buffer_names")` check? This attribute should always be defined?

Reply:
you are right. we do not need hasattr check. removed the check